packages/material-ui/src/AccordionSummary/AccordionSummary.js

```diff
@@ -1,74 +1,126 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
+import { deepmerge } from '@material-ui/utils';
+import { unstable_composeClasses as composeClasses } from '@material-ui/unstyled';
+import experimentalStyled from '../styles/experimentalStyled';
+import useThemeProps from '../styles/useThemeProps';
 import ButtonBase from '../ButtonBase';
-import withStyles from '../styles/withStyles';
 import AccordionContext from '../Accordion/AccordionContext';
+import accordionSummaryClasses, {
+  getAccordionSummaryUtilityClass,
+} from './accordionSummaryClasses';
 
-export const styles = (theme) => {
+const overridesResolver = (props, styles) => {
+  return deepmerge(styles.root || {}, {
+    [`& .${accordionSummaryClasses.content}`]: styles.content,
+    [`& .${accordionSummaryClasses.expandIconWrapper}`]: styles.expandIconWrapper,
+  });
+};
+
+const useUtilityClasses = (styleProps) => {
+  const { classes, expanded, disabled } = styleProps;
+
+  const slots = {
+    root: ['root', expanded && 'expanded', disabled && 'disabled'],
+    focusVisible: ['focusVisible'],
+    content: ['content', expanded && 'expanded'],
+    expandIcon: ['expandIcon', expanded && 'expanded'],
```

Comment:
```suggestion
    expandIconWrapper: ['expandIconWrapper', expanded && 'expanded'],
```

Looks like this name was wrong.